gitlab/utils.py

```diff
@@ -57,7 +57,7 @@ def copy_dict(dest: Dict[str, Any], src: Dict[str, Any]) -> None:
 
 
 def clean_str_id(id: str) -> str:
-    return quote(id, safe="")
+    return urllib.parse.quote(id, safe="")
```

Comment:
Just thinking: does it still make sense to call this `clean_str_id`? I think we could call it `urlencode()` or so? or maybe some other name that doesn't clash with https://docs.python.org/2/library/urllib.html#urllib.urlencode but applies more generally to what we do with it now.

Reply:
Done. changed to `url_encode`